cni/network/network.go

```diff
@@ -72,6 +72,10 @@ const (
 	Baremetal ExecutionMode = "baremetal"
 )
 
+const (
+	rehydrateNetworkInfoOnReboot = false
+)
```

Comment:
why do we need a constant `false`? this just hides whatever the real test is from me when I read the code where it is checked (like, is this actually a "is running on windows" bit?)
I think it would be much clearer if, where ever we use this conditional, we were checking the actual underlying condition that makes this switch

Reply:
nope this is not running on windows.. cnm and cni uses same code path. have to rehydrate for cnm but not for cni